src/monobase/util.py

```
@@ -173,12 +173,15 @@ def parse_requirements(req: str) -> dict[str, str | Version]:
         if line == '' or line.startswith('#') or line.startswith('--'):
             continue
         if '==' in line:
-            parts = line.split('==')
+            parts = line.split('==', 1)
```

Do we need to support this? A dependency like `a==b==c` feels like a typo to me - maybe it's better to throw an error?

---

This is only ever used for parsing frozen requirements from either `uv pip freeze` or `uv pip compile` so it should be only one of these based on my trial & error...:
* `<pkg>==<resolved version>`
* `<pkg> @ <URL>`
* `<path>`
The path one could be ambiguous like `./requests` vs `requests` and the spec seems unclear? Anyway hence the `os.path.exists(line)` check here. 
https://pip.pypa.io/en/stable/reference/requirements-file-format/

---

Forgot to mention it's for the `@` case e.g. `requests@https://github.com/.../requests@<hash>`, so yeah probably won't see such thing for `==` anyway. But then the version is passed as is to the re-constructed `requirements.txt` into `uv pip install` which should catch any typos.